### DISCUSSION TITLE
set steps (speed, Quality ...)

### DISCUSSION THREAD

**Discussion by kalle07** [lllyasviel/Fooocus#3175] [Q&A] [answered]
can i set the steps for Performance in config.txt ?
and how ?

**Comment by Rogala** [comment, marked as answer]
![image](https://github.com/lllyasviel/Fooocus/assets/7437582/ccfbafbe-54d7-4aa7-b4ad-585587723ebd)

**Reply by kalle07** [reply]
i see... thx

but i want set by my self  
for high speed 20
for quality 40
maybe for highquality 100

**Reply by mashb1t** [reply]
you can't do this in the config, as these values are in an enum. Feel free to adjust the code though, see https://github.com/lllyasviel/Fooocus/blob/cef92ffffe2aa8669b9284309821279cf22cd996/modules/flags.py#L131-L135

The only thing you can do is to create a preset for each of the performances overriding the steps with `"default_overwrite_step": 30` (30  => your custom step count).

**Reply by kalle07** [reply]
thx !!!

**Reply by AnalyticsXXI** [reply]
By the way, how to disable downloading default checkpoint on launching app?

**Reply by mashb1t** [reply]
@AnalyticsXXI you can create a custom preset (e.g. based on the default one), modify the model in it and start Fooocus with `--preset xyz` to use a custom model instead of Juggernaut v8.
